packages/flutter_tools/lib/src/reporting/usage.dart

```diff
@@ -55,6 +55,8 @@ enum CustomDimensions {
   commandBuildAppBundleBuildMode, // cd42
   buildEventError,  // cd43
   commandResultEventMaxRss,  // cd44
+  commandRunAndroidEmbeddingVersion, // cd45
```

Comment:
Might need some help here. I don't know how to add these.

Reply:
I can add them once you submit this PR.

Reply:
thanks!

Reply:
I didn't find them in GA. Did you add them?

Reply:
Sorry. I just added. It should be possible to retrieve the data even if the custom dimension isn't registered, since GA just uses the index instead of the name to store the data.

Reply:
ummm doesn't seem queriable https://plx.corp.google.com/scripts2/script_5d._cc5d93_0000_2c04_a547_94eb2c112722

Wonder if the GA definition affects BigQuery exporting.